collectors/net.go

```diff
@@ -168,6 +170,16 @@ func stringInSlice(str string, list []string) bool {
 		if v == str {
 			return true
 		}
+
+		if strings.HasPrefix(strings.TrimSpace(v), "~") {
+
+			matched, _ := regexp.MatchString(strings.Replace(strings.TrimSpace(v), "~", "", 1), str)
+
```

Comment:
`return matched`

Reply:
Nope :) return matched will not process other elements of to compare list. You can only return matched in case of success ;)